PxWeb/app.config

```diff
@@ -3,5 +3,6 @@
 
 	<appSettings>
 		<add key="dbconfigFile" value="SqlDb.config" />
+		<!--<add key="log4net.Internal.Debug" value="true"/>-->
```

Comment:
Maybe we should add a comment to this? 
"Remove comment to enable log4net logging"

Reply:
Sound good I will add it.